api/distributedContext/context.go

```diff
@@ -12,21 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package tag
+package distributedContext
 
 import (
 	"context"
 	"runtime/pprof"
 )
 
-type ctxTagsType struct{}
+type ctxEntriesType struct{}
 
 var (
-	ctxTagsKey = &ctxTagsType{}
+	ctxEntriesKey = &ctxEntriesType{}
 )
 
 func WithMap(ctx context.Context, m Map) context.Context {
-	return context.WithValue(ctx, ctxTagsKey, m)
+	return context.WithValue(ctx, ctxEntriesKey, m)
```

Comment:
Since this is in a package named "distributedcontext" (or "dctx"), the type name should be "EntriesKey"